papers/ring_vrf/pedersen_vrf.tex

```diff
@@ -56,41 +60,50 @@ \section{Ring VRF construction}% {Ring VRFs from the Pedersen VRF}
 These handle a Pedersen commitment to the secret key \sk,
  together with its opening \openpk.
 \begin{itemize}
-\item $\PedVRF.\CommitKey(\pk)$ \,
+\item $\PedVRF.\CommitKey(\sk)$ \, %I changed it because we commit to sk
 returns a blinding factor $\openpk \leftsample \F_p$
-and a commitment $\compk = \pk + \openpk \, \genB$.
+and a commitment $\compk = \sk \genG + \openpk \, \genB$.
 \item $\PedVRF.\OpenKey(\compk,\openpk)$ \,
 returns $\pk = \compk - \openpk \, \genB$.
 \end{itemize}
 % \item $\PedVRF.\CommitKey$ selects a blinding factor $\openpk$ uniformly
 %  at random from $\F_p$ and computes the commitment $\compk = \pk + \openpk \, \genB$.
 % \item $\PedVRF.\OpenKey$ just returns $\pk = \compk - \openpk \, \genB$.
-These hide their public key argument \pk too, but alone do not bind \pk.
-A successful $\PedVRF.\Verify$ below proves knowledge of \compk,
- which makes the commitment binding.
+
+%These hide their public key argument \pk too, but alone do not bind \pk.
+%A successful $\PedVRF.\Verify$ below proves knowledge of \compk,
+ %which makes the commitment binding. % COMMENT Handan: I don't think this statement is true because Sign does not bind to pk. because there is nothing here that shows that the committed secret key multiplied by G is pk.
+Now, we introduce how to generate a proof for the correct evaluation of an input which is generated by the committed secret key. 
+
 
 \begin{itemize}
 \item $\PedVRF.\Sign : (\sk,\openpk,\msg,\aux) \mapsto \sigma$ \,
     % takes a secret key \sk and blinding factor \openpk, an input $\msg$, and associated data \aux, and then performs
-    First computes $\In := H_{\grE'}(\msg)$ and $\PreOut := \sk \, \In$.
+    It generates a Chaum-Pedersen DLEQ proof  showing the following relation by running the algorithm $ \NIZK_{\mathcal{R}_{eval}}.\Prove(((\genG, \genB,\grE,\compk,\PreOut,\In); (x, \openpk))) $ which outputs $ \pi_{eval}$ and returns a signature $ \sigma = (\PreOut,\compk,\pi_{eval}) $
```

Comment:
Instead of "and returns a signature" I would say "; finally, it returns ..."
Also, I believe the witness is (\sk, \openpk) instead of (x, \openpk).